mappings/net/minecraft/client/util/Allocator.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/class_301 net/minecraft/client/util/Allocator
+	FIELD field_1643 ALLOCATOR_UNTRACK Ljava/lang/invoke/MethodHandle;
+	METHOD method_1406 untrack (Lorg/lwjgl/system/Pointer;)V
```

Comment:
Missing parameter name should be `pointer`. Note that it may display as `pointer` in Enigma but this is just an inferred name, and the Loom toolchain will infer it differently, producing `pointer_1` if unmapped here.

Reply:
Ah. Thank you! I'll have to keep that in mind. I've been not mapping parameters in some cases because the value Enigma shows is already correct.